test/unit_tests/forest/forest_test.cpp

```diff
@@ -40,22 +40,22 @@ BOOST_AUTO_TEST_CASE(forest_int_iterator_test) {
     forest<int>::iterator begin = f.begin(), end = f.end();
     (void)begin;
     (void)end;
-    boost::function_requires<boost::BidirectionalIteratorConcept<forest<int>::iterator>>();
-    boost::function_requires<boost::BidirectionalIteratorConcept<forest<int>::const_iterator>>();
-    boost::function_requires<boost::BidirectionalIteratorConcept<forest<int>::preorder_iterator>>();
-    boost::function_requires<
+    std::function_requires<boost::BidirectionalIteratorConcept<forest<int>::iterator>>();
```

Comment:
boost::function_requires is part of boost concept checks, there is no direct std:: equivalent. This code will not compile.